frame/aura/src/mock.rs

```diff
@@ -86,26 +85,25 @@ impl pallet_timestamp::Config for Test {
 	type WeightInfo = ();
 }
 
-thread_local! {
-	static DISABLED_VALIDATORS: RefCell<Vec<AuthorityIndex>> = RefCell::new(Default::default());
+parameter_types! {
+	static DISABLED_VALIDATORS: Vec<AuthorityIndex> = Default::default();
```

Comment:
Couldn't convert this to CamelCase because another variable uses the name. What could this be renamed to? @kianenigma

Reply:
Just use some other name, like `DisabledValidatorTestValue`